### PR TITLE
Phone metrices-webapp module three capstone project

### PR DESCRIPTION
**Pull request by juaryimami**
### This React capstone project :-
---
I have selected an API that provides numeric data about Latin american air pollution and then build the web-app around it. The web-app will have two pages:
###  Home page
---
-  When the page loads, the web-app shows the list of categories that could be filtered out by some parameter, for example by the category name.
- Along with the category name, you will display some numeric values per category that come from the API.
- When the user clicks (or taps) on a category item, the application navigates to the details page.
- I built a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux.
### Details page
---
- In the details page, the web-app retrieves data from the API to show detailed data in the category.
- When the user clicks on the "Back" button (<), the user navigates to the home page.
---
#### Thank you for reviewing my project!!!!!!.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,7 +196,7 @@ Give a ⭐️ if you like this project!
 
 - **How to use the project**
 
-  - ```git clone https://github.com/juaryimami/math-magicians```
+  - ```git clone https://github.com/juaryimami/metrics-webapp```
 
 - **How to contribute to the project**
```
Comment (ishimwezachee) [inline README.md:205]: - [ ]  In your Readme file, in the Acknowledgements part, you should give appropriate credit to Nelson Sakwa the author of the original design, as required by the Creative Commons License.

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @juaryimami , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
image


## Suggested changes
_Check the comments under the review._

 - [ ] Your project should have a video presentation full fill the following 
- summarize the features of the project 
- less than 5 min long 
- clear description of the project purpose 
- Piece of code that you are proud of 

- [ ]  You should write a well detailed PR, this will help the reviewers to an ideas of what you've worked on before they start review your project. [Here is the benefits of good pull request description](https://formidable.com/blog/2020/good-pr-descriptions/#:~:text=PR%20descriptions%20make%20you%20a,a%20better%20programmer%20and%20teammate.)

 [ OPTIONAL] I would recommend you to add a simple border to each country to prevent the confusion a user might face while click to to specific country to check out its information. 
 
image




You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (juaryimami) [reply README.md:205]: ok I will do it soon. Thank you so much sir!!!

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @juaryimami 😊, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

Almost near

 ## Highlights
✔️ There is no linter error present
✔️ Good presentation demo

 ## Required Changes ♻️
- [ ] As stated by the previous reviewer, kindly consider writing a good descriptive pull request which contains what you made in your project. You can refer to this [PR](https://github.com/omarsalem7/space-app/pull/34#:~:text=This%20pull%20request,the%20reserved%20rockets.)

## Optional suggestions

- [ ] [OPTIONAL] For the next capstone project, kindly consider using the correct gitflow by open the PR from the dev branch to the main branch.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢

Hi @juaryimami,
 
Good job on fixing all changes highlighted by the previous reviewers 👍🏽

Your project is complete!  

## Highlights:

✅ You used Gitflow correctly 👍🏽
✅ LInter checks passed 👍🏽
✅ Good descriptive PR 👍🏽
✅ App is hosted correctly and works fine as expected 👍🏽

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**